pybotvac/account.py

```diff
@@ -24,6 +24,30 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+USER_SCHEMA = Schema(
+    {
+        Required("id"): str,
+        "first_name": Any(str, None),
+        "last_name": Any(str, None),
+        "company": Any(str, None),
+        "locale": Any(str, None),
+        "phone_number": Any(str, None),
+        "street_1": Any(str, None),
+        "street_2": Any(str, None),
+        "city": Any(str, None),
+        "post_code": Any(str, None),
+        "province": Any(str, None),
+        "state_region": Any(str, None),
+        "country_code": Any(str, None),
+        "source": Any(str, None),
+        "developer": Any(bool, None),
+        Required("email"): str,
```

Comment:
Using my account, the returned json doesn't have an "email" key.

Simone

```suggestion
        "email": Any(str, None),
```